senders/client_factory.go

```diff
@@ -219,14 +222,8 @@ func SDKMetricsTags(tags map[string]string) Option {
 	// prevent caller from accidentally mutating this option.
```

Comment:
Can you update the documentation for this function? It says:

> By default, the sender does not attach any tags to internal metrics.

But it attaches a `pid` and `version` tag by default now.

Reply:
Nice catch.  Done.